content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,230 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+   parameters.
+
+.. spoiler::
+   An example tutorial for the Epson TM-U220B printer model.
+
+   The following is an example of the troubleshooting process for a TM-U220B printer model to use
+   the `ESC *` command. The receipt picture below is an example of a receipt that is printing
+   correctly due to proper formatting (in theory):
+
+   .. image:: troubleshooting/receipt-example.png
+      :align: center
+      :alt: Properly formatted receipt picture from a demo database.
+
+   Trying to print this receipt right-away prior to the proper formatting  will not work as the
+   TM-U220B printer model does not support `GS v 0`. Instead random characters will print:
+
+   .. image:: troubleshooting/receipt-print-random-letters.png
+      :align: center
+      :alt: Printer paper with seemingly random characters.
+
+   To properly configure formatting for the Epson TM-U220B printer model take the following steps.
+
+   After consulting Epson's website for compatibility for both of the commands: `GS v 0` and
+   `ESC *`, it can be seen that indeed the TM-U220B is not compatible with `GS v 0`, but is
+   compatible with `ESC *`.
+
+   .. image:: troubleshooting/epson-compatibility-compare.png
+      :align: center
+      :alt: Epson compatibility evaluation from Epson website.
+
+   When adding the printer, *CUPS* will ask which printer should be added:
+
+   .. image:: troubleshooting/add-printer.png
+      :align: center
+      :alt: Administration menu, add printer selection.
+
+   In this case, the printer is connected via :abbr:`USB (Universal Serial Bus)` so it won' be part
+   of the :guilabel:`Discovered Network Printers`. Instead it is likely part of the
+   :guilabel:`Unknown` selection. By unplugging the printer's :abbr:`USB (Universal Serial Bus)`
+   cable from the :abbr:`IoT (Internet of Things)` box and refreshing the page, the
+   :guilabel:`Unknown` printer disappears. By plugging it back in, then it reappears, so it can be
+   said that this is the printer in question.
+
+   For the naming convention, since it will need to print using the `ESC *` command, it is
+   imperative to add `__IMC`. Reference the printer model on `Epson's ESC * site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ to find out more
+   about the *density* parameters.
+
+   .. image:: troubleshooting/epson-TM-U220-specification.png
+      :align: center
+      :alt: Epson TM-U220 specifications on manufacturers website.
+
+   For this particular model, TM-U220, `m` should be equal to 0 or 1. While referencing the
+   :guilabel:`Description` below the pink box in the above picture, the `m` values could be 0, 1, 32
+   or 33. So in this printers case, the `m` value can NOT be 32 or 33 (otherwise there will be
+   random characters printed).
+
+   If we check the table for the values: 32 and 33, they both occur if the :guilabel:`Number of bits
```

Comment:
We don't use pronouns like "we" in docs :astonished: Can you rewrite all instances of this, please? (:

Reply:
Noted, this is the only instance. Thank you.